embassy-stm32/src/rcc/l0.rs

```diff
@@ -250,7 +250,7 @@ pub(crate) unsafe fn init(config: Config) {
     BackupDomain::configure_ls(
         config.rtc.unwrap_or(RtcClockSource::NOCLOCK),
         config.lsi,
-        config.lse.map(|_| Default::default()),
+        config.lse.map(|_| LseCfg::Oscillator(Default::default())),
```

Comment:
None of the other rcc modules should be changed; the default should be appropriately set in bd.rs.